### PR TITLE
added  documentation for monorepo code quality analysis

### PR DESCRIPTION
**Pull request by surajs01**
-added documentation

### CODE REVIEW

File: Writerside/topics/Java.md [hunk 1]
```diff
@@ -150,22 +150,18 @@ Add this to your workflow file to run Astronuts Code Quality Checks on your Java
...
         uses: astronuts-app/astronuts-code-quality-action@v4
-        with:
-          sourceLanguage: 'java'
-          buildSystem: 'maven'
       </code-block>
     </tab>
 </tabs>
-
+Note* : There is no need to specify build systems and languages used in the project as we use our auto detection system to detect that.  
+        -> For now we only support test written with JaCoCo library to generate reports.
```
Comment (surajs01) [inline Writerside/topics/Java.md]: fix documantation

File: Writerside/topics/Monorepo.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+              - Build-Typescript
+            runs-on: ubuntu-latest
+            steps:
+              - uses: actions/checkout@v4
+              - name: Run Astronuts Code Quality Checks
+                uses: astronuts-app/astronuts-code-quality-action@v4
+        </code-block>
+    </tab>
+</tabs>
+Note* : There is no need to specify build systems and languages used in the project as we use our auto detection system to detect that.
```
Comment (surajs01) [inline Writerside/topics/Monorepo.md]: remove this

File: Writerside/topics/Python.md [hunk 3]
```diff
@@ -54,10 +54,10 @@ Step 3 : Run Astronuts Code Quality Checks on your Python Project.
 ```bash
       - name: Run Astronuts Code Quality Checks
         uses: astronuts-app/astronuts-code-quality-action@v4
-        with:
-          sourceLanguage: 'python'
-          buildSystem: 'pip'
 ```
+Note* : There is no need to specify build systems and languages used in the project as we use our auto detection system to detect that.
```
Comment (surajs01) [inline Writerside/topics/Python.md]: remove this

File: Writerside/topics/Typescript.md [hunk 4]
```diff
@@ -45,10 +45,10 @@ Step 2 : Run Astronuts Code Quality Checks on your Python Project.
 ```bash
 - name: Run Astronuts Code Quality Checks
   uses: astronuts-app/astronuts-code-quality-action@v4
-  with:
-    sourceLanguage: 'typescript'
-    buildSystem: 'npm'
 ```
+Note* : There is no need to specify build systems and languages used in the project as we use our auto detection system to detect that.
```
Comment (surajs01) [inline Writerside/topics/Typescript.md]: remove this

**Review by arunkpatra-astro — APPROVED** [review]

**Pull request merged**